app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,108 @@
+# TASKS = ["Do laundry", "Wash dogs", "Buy new sheets"]
+
+class TasksController < ApplicationController
+  def index
+    @tasks = Task.all
+  end
+
+  def show
+    task_id = params[:id]
+    @task = Task.find_by(id: task_id)
+    if @task.nil?
+      # head :temporary_redirect
+      redirect_to root_path
+      return
+    end
+  end
+
+  # def new
+  #   @task = Task.new
+  # end
```

Comment:
I'm not sure why this is commented out.  It breaks creating a task in the browser.  (Uncommenting fixes the issues though.)